docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -216,6 +286,9 @@ In future shells, you can activate and use this environment with:
 
 See the `workflow-tools repository <https://github.com/ufs-community/workflow-tools>`__ for additional documentation. 
```

Comment:
Didn't see this before, but lines 280-287 need to be indented 3 spaces to fall within section 2).